openpgp/packet/signature.go

```diff
@@ -1260,6 +1270,10 @@ func (sig *Signature) buildSubpackets(issuer PublicKey) (subpackets []outputSubp
 		subpackets = append(subpackets, outputSubpacket{true, featuresSubpacket, false, []byte{features}})
 	}
 
+	if sig.Exportable != nil && *sig.Exportable {
+		subpackets = append(subpackets, outputSubpacket{true, exportableCertSubpacket, true, []byte{1}})
+	}
```

Comment:
This is not complete, since the default value of "exportable" is true, the subpacket should also be added if `*sig.Exportable` is false.

Reply:
Ah, I missed that. If I understood correctly the absence of the subpacket indicates that the certification is exportable. Then, I guess, it will be okay to add the subpacket when we have a `false` value for `*sig.Exportable`?

Reply:
Yeah, I guess so :)